lib/get_user_move.ex

```diff
@@ -0,0 +1,60 @@
+import WinningCombo
+
+defmodule GetUserMove do
+  def getMove(board, token) do
+    raw_input = _slugInput("Your Turn! Please choose an empty game space. ")
+    checked_input = _inputChecker(
+            _createValidInputArray(board, token, []), raw_input)
+
+    _replacementFunction(board, token, checked_input, [])
+
+  end
+
+  def _slugInput(messagePrompt) do
+    input = IO.gets(messagePrompt)
+    if input == :eof do
+      raise "Why wont this work"
```

Comment:
Did you mean to leave this here?

Reply:
The :eof error occasionally pops up when mocking. I can change the error message to something a little more formal.

Reply:
error now raises: "End of File! Check for propper mocks and imports"